python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -556,12 +567,9 @@ def read_from_stream(
             filenames = [str(f) for f in filenames]
             filenames += ['psf', 'psf_shape']
         else:
-            filenames = None
+            return NotImplemented("It is inefficient to read the entire archive from a stream.")
```

Comment:
```suggestion
            return NotImplemented
```

suggests a problem with test coverage:

```python
>>> NotImplemented("It is inefficient to read the entire archive from a stream.")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NotImplementedType' object is not callable
```

Reply:
I must not understand how this works. From my understanding https://github.com/lsst/meas_extensions_scarlet/blob/main/tests/test_deblend.py#L386-L389 should be testing this.

Reply:
Also, this ran in ci_hsc, so I'm surprised that it didn't cause an issue there...